webpack/JobWizard/autofill.js

```diff
@@ -39,10 +39,14 @@ export const useAutoFill = ({
             handleSuccess: ({ data }) => {
               setSelectedTargets(currentTargets => ({
                 ...currentTargets,
-                hosts: (data.results || []).map(({ id, name }) => ({
-                  id,
-                  name,
-                })),
+                hosts: (data.results || []).map(
+                  // eslint-disable-next-line camelcase
+                  ({ id, name, display_name }) => ({
+                    id,
+                    // eslint-disable-next-line camelcase
+                    name: display_name || name,
+                  })
+                ),
```

Comment:
@ShimShtein I found that the reason the test is failing is that we were missing the `name` param here (and not the graphql as we thought).
If this fixes the issue, we can discuss whether we want to apply other changes.